.gitignore

```diff
@@ -8,7 +8,6 @@ out/
 
 # Logs
 logs
-*.log
```

Comment:
@DatGuyJonathan - Could we be more specific and just ignore the *.log from certain folders? You can use nested .gitignore files. See https://stackoverflow.com/questions/3305869/are-multiple-gitignores-frowned-on

I worry that if we just whitelist *.log that someone will accidentally add and commit a *.log file.